packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -174,10 +176,19 @@ export async function getHistory(
   const { org } = getOrgFromReq(req);
   const { type, id } = req.params;
 
-  const events = await Promise.all([
-    findByEntity(org.id, type, id),
-    findByEntityParent(org.id, type, id),
-  ]);
+  let events = [];
+  if (type === "savedGroup") {
+    // saved groups are a special case because using an ID here doesn't make sense.
```

Comment:
This doesn't seem specific to saved groups.  Just anytime we want to show an audit log on a list page instead of a detail page.  Seems like that should maybe be a different API endpoint.  `/history/:type/:id` is the current one and we can add `/history/:type` as a new endpoint

Reply:
done